### ISSUE TITLE
Full Accessibility & 508 Office Audit [Authenticated Experience - My VA 2.0]

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#28306]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(Platform understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [x] VFS: Using the Slack Workflow ([User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)), notify Collaboration Cycle team of this Github ticket in #vfs-platform-support. 
- [x] Platform: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)
- [ ] VFS: Close this ticket when all steps above are completed.

## Artifacts
- Feature in staging: [staging.va.gov/my-va](https://staging.va.gov/my-va)
- [My VA Dashboard 2.0 staging info](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-myva-passwords.md)
- [Test plan in TestRail](https://dsvavsp.testrail.io/index.php?/plans/view/1799)
- [Test cases in TestRail](https://dsvavsp.testrail.io/index.php?/suites/view/5&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=1069)

## Additional information
As part of this review, Platform will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Platform Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by ajakabcin** [comment]
@shiragoodman @noahgelman and @1Copenut - I created this issue to get things pulled together for documentation purposes, but we're still working through a few lingering defects from our original accessibility review. We're not officially ready to have this sent to for the audit as we have some minor issues still open. I'll comment on the ticket when it's ready to proceed to the next step.

Thanks!

**Comment by shiragoodman** [comment]
thanks @ajakabcin ! Tagging @joshkimux and @AngelaFowler82 for awareness.

**Comment by ajakabcin** [comment]
Ok @shiragoodman we're good to go on this now. Thanks so much!

cc @joshkimux and @AngelaFowler82

**Comment by AngelaFowler82** [comment]
Looking at user+36, it shows that the user has applied for disability compensation and educational benefits, both of which appear in the "Benefits you might be interested section." Is there any way to make the content of "Benefits you might be interested in" more dependent on the benefits the user has applied for?

**Comment by ajakabcin** [comment]
Hi @AngelaFowler82 - yes we've noticed this, and unless I'm misunderstanding user 36's situation, we have a ticket in our backlog to make this change. I believe the logic currently looks for "active" benefits vs applications in progress to determine whether to show that section or not. Erik caught this earlier this summer and made the recommendation that we adjust that logic to include apps in progress as well as a Veteran actively receiving benefits as scenarios when we should hide the "Benefits you might be interested in" section. See #26652

**Comment by joshkimux** [comment]
@ajakabcin Big shoutout to the auth exp team for making this one of the smoothest full 508 audits in memory. We've wrapped this up in record time!

@1Copenut @noahgelman this full audit is complete and is ready for y'alls final review 😄

**Comment by ajakabcin** [comment]
Closed as complete
